tools/generate_torch_version.py

```diff
@@ -60,6 +60,15 @@ def get_torch_version(sha: Optional[str] = None) -> str:
         if sha is None:
             sha = get_sha(pytorch_root)
         version += "+git" + sha[:7]
+    if os.getenv("NVIDIA_PYTORCH_VERSION"):
+        # see if it matches YY.MM
+        str_nvidia_pytorch_version = os.getenv("NVIDIA_PYTORCH_VERSION")
+        pattern = re.compile(r'^\d{2}\.\d{2}$')
+        if bool(pattern.match(str_nvidia_pytorch_version)):
+          version += ".nv" + str_nvidia_pytorch_version
+        else:
+          version += ".nvInternal"
```

Comment:
How about passing it explicitly via `PYTORCH_VERSION_SUFFIX`

Reply:
As discussed, I will think more about it and re-open when it is ready.